src/julia-parser.scm

```diff
@@ -68,7 +88,9 @@
                              (pushprec (cdr L) (+ prec 1)))))
                      (pushprec (map eval prec-names) 1)
                      t))
-(define (operator-precedence op) (get prec-table op 0))
+(define (operator-precedence op) (get prec-table
+                                      (maybe-strip-op-suffix op)
```

Comment:
I was hoping that `maybe-strip-op-suffix` would be fast enough, because in the common case `strip-op-suffix` is a no-op and `no-suffix?` isn't even called.

Is there any way to benchmark the impact of this?